independent-publisher-connectors/SHRTCODE/apiDefinition.swagger.json

```diff
@@ -0,0 +1,189 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SHRTCODE",
+    "description": "With this free Link Shortener you can make Links shorter and easier to remember.",
+    "version": "1.0",
+    "contact": {
+      "name": "Chandra Sekhar",
+      "url": "https://github.com/sekharmalla",
+      "email": "chandra.malla@msftcommunity.com"
+    }
+  },
+  "host": "api.shrtco.de",
+  "basePath": "/v2/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/shorten": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "succcess",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "ok": {
+                  "type": "boolean",
+                  "description": "ok",
+                  "title": "OK",
+                  "enum": [true, false]
+                },
+                "result": {
+                  "type": "object",
+                  "properties": {
+                    "code": {
+                      "type": "string",
+                      "description": "code",
+                      "title": "Code"
+                    },
+                    "short_link": {
+                      "type": "string",
+                      "description": "short_link",
```

Comment:
@sekharmalla descriptions should look like valid sentences. You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).